src/frontend/src/pages/ChangeRequestsPage/ChangeRequests.tsx

```diff
@@ -8,14 +8,21 @@ import { routes } from '../../utils/routes';
 import ChangeRequestDetails from '../ChangeRequestDetailPage/ChangeRequestDetails';
 import ChangeRequestsTable from './ChangeRequestsTable';
 import CreateChangeRequest from '../CreateChangeRequestPage/CreateChangeRequest';
+import ChangeRequestsOverview from './ChangeRequestsOverview';
+import ChangeRequestsTitle from './ChangeRequestsTitle';
 
 const ChangeRequests: React.FC = () => {
   return (
-    <Switch>
-      <Route path={routes.CHANGE_REQUESTS_NEW} component={CreateChangeRequest} />
-      <Route path={routes.CHANGE_REQUESTS_BY_ID} component={ChangeRequestDetails} />
-      <Route path={routes.CHANGE_REQUESTS} component={ChangeRequestsTable} />
-    </Switch>
+    <>
+      <ChangeRequestsTitle />
```

Comment:
This will show up on screen we dont want it to show up on now. We only want it to show up on overview page and all page, not on create CR or cr details. Look at how we did the task list tab panel thing.